### PR TITLE
Added gradient to hero image

### PR DESCRIPTION
**Pull request by jessgilbert**
image

### CODE REVIEW

File: core/sass/pages/capital-invest/_capital-invest-landing-page.scss [hunk 1]
```diff
@@ -5,6 +5,21 @@
 @import '../../partials/mixins';
 .capital-invest-landing-page {
 
+  .hero {
+
+    position: relative;
+
+    &:before {
+    background-image: linear-gradient(to left, transparent 20%, rgba(0,0,0.7,0.7) 100%);
```
Comment (eadpearce) [inline core/sass/pages/capital-invest/_capital-invest-landing-page.scss]: super nitpicky - can you indent lines 13-19 one more level pls

File: core/sass/main.css [hunk 2]
```diff
@@ -322,6 +322,18 @@ body.js-enabled .landing-page .country-selector-dialog select {
   padding-left: 60px;
 }
 
+.capital-invest-landing-page .hero {
+  position: relative;
```
Comment (eadpearce) [inline core/sass/main.css]: i'm not sure why this file exists here. main.css already exists in core/static/styles. was this made by the gulp task?

File: core/sass/pages/capital-invest/_capital-invest-landing-page.scss (hunk 1 above)
Comment (jessgilbert) [reply core/sass/pages/capital-invest/_capital-invest-landing-page.scss]: Oops missed that, done :)

File: core/sass/main.css (hunk 2 above)
Comment (jessgilbert) [reply core/sass/main.css]: Yes it was

File: core/sass/pages/capital-invest/_capital-invest-landing-page.scss [hunk 3]
```diff
@@ -5,6 +5,21 @@
 @import '../../partials/mixins';
 .capital-invest-landing-page {
 
+  .hero {
```
Comment (MichaelWalker) [inline core/sass/pages/capital-invest/_capital-invest-landing-page.scss]: I wonder if this should really just be applied to all hero banners.

@eadpearce @jessgilbert, what do you think about putting this in directory-components?

File: core/sass/main.css (hunk 2 above)
Comment (eadpearce) [reply core/sass/main.css]: how weird. i'll see if i can find out why. different PR though

File: core/sass/pages/capital-invest/_capital-invest-landing-page.scss (hunk 3 above)
Comment (eadpearce) [reply core/sass/pages/capital-invest/_capital-invest-landing-page.scss]: we only really need this to increase contrast on banners that don't have the blue box for text. we could add it as a utility class however. that'd be neat. maybe 4 different classes for each direction of gradient?

Comment (MichaelWalker) [reply core/sass/pages/capital-invest/_capital-invest-landing-page.scss]: They are different templates anyway though now aren't they?

Is it as simple as;
hero -> should NOT have the overlay

Comment (eadpearce) [reply core/sass/pages/capital-invest/_capital-invest-landing-page.scss]: in theory that should be the case :'D i spoke with michal though and i think we gotta do it on a case-by-case basis as it all depends on whether the image/text needs it or not. we were originally going to add it to case studies but then decided against it. so i think hold off for now

**Pull request closed without merging**